bosh_aws_cpi/lib/cloud/aws/stemcell_creator.rb

```diff
@@ -120,11 +126,10 @@ def image_params(snapshot_id)
       params.merge!(
         :name => "BOSH-#{SecureRandom.uuid}",
         :architecture => architecture,
-        :block_device_mappings => params[:block_device_mappings].merge(
-          default_ephemeral_disk_mapping
-        )
       )
 
+      params[:block_device_mappings].push(*default_ephemeral_disk_mapping)
```

Comment:
I was getting the following error when I first tried using `merge` on the same line. Probably some ruby n00b mistake, so I switched to this but I imagine there might be a more correct solution.

     NoMethodError:
       undefined method `merge' for [{:device_name=>"/dev/xvda", :ebs=>{:snapshot_id=>"id"}}]:Array